spikeextractors/extractors/neoextractors/neobaseextractor.py

```diff
@@ -129,6 +129,10 @@ def get_sampling_frequency(self):
 
     def get_channel_ids(self):
         chan_ids = self.neo_reader.header['signal_channels']['id']
+
+        # force chan_ids to be int (new neo default=str)
+        chan_ids = np.array([np.int(i) for i in chan_ids])
```

Comment:
@sbuergers NEO now allows any string to be a channel id. I think that this would fail if the channel name is for example `'channel #0'`. I would just make the channel_ids `np.arange(len(chan_ids))` for the time being. 

Note that in the new SI refractor (https://github.com/SpikeInterface/spikeinterface/pull/88) we will allow channel ids to be int or strings.